### ISSUE TITLE
[terra-status] Missing themeable variables

### ISSUE THREAD

**Issue by ryanthemanuel** [cerner/terra-core#3052]
# Bug Report

The following themeable variables are missing (we should theme them even though they're dev site only for example purposes):

**clinical-lowlight-theme**

--terra-status-attention-border-left-color - packages/terra-status/lib/terra-dev-site/doc/example/colors.module.scss (line: 9, column: 4)
--terra-status-info-border-left-color - packages/terra-status/lib/terra-dev-site/doc/example/colors.module.scss (line: 17, column: 4)
--terra-status-success-border-left-color - packages/terra-status/lib/terra-dev-site/doc/example/colors.module.scss (line: 13, column: 4)

**orion-fusion-theme**

--terra-status-attention-border-left-color - packages/terra-status/lib/terra-dev-site/doc/example/colors.module.scss (line: 9, column: 4)
--terra-status-info-border-left-color - packages/terra-status/lib/terra-dev-site/doc/example/colors.module.scss (line: 17, column: 4)
--terra-status-success-border-left-color - packages/terra-status/lib/terra-dev-site/doc/example/colors.module.scss (line: 13, column: 4)

**Comment by neilpfeiffer** [comment]
For the Lowlight theme:
```
:local {
  .clinical-lowlight-theme {
    /*{ ...existing styles here...}*/

    // Used for docsite custom color examples only
    --terra-status-attention-border-left-color: #f94e4e;
    --terra-status-info-border-left-color: #5a81ff;
    --terra-status-success-border-left-color:  #42af00;
  }
}
```

----

For the Fusion theme:
```
:local {
  .orion-fusion-theme {
    /*{ ...existing styles here...}*/

    // Used for docsite custom color examples only 
    --terra-status-attention-border-left-color: #c00;
    --terra-status-info-border-left-color: #007cc3;
    --terra-status-success-border-left-color:  #78c346;
  }
}
```

**Comment by emilyrohrbough** [comment]
Resolved in https://github.com/cerner/terra-core/pull/3107
